src/chains/ethereum/options/src/logging-options.ts

```diff
@@ -92,23 +99,48 @@ export const LoggingOptions: Definitions<LoggingConfig> = {
     cliAliases: ["q", "quiet"],
     cliType: "boolean"
   },
-  logger: {
-    normalize,
-    cliDescription:
-      "An object, like `console`, that implements a `log` function.",
-    disableInCLI: true,
-    // disable the default logger if `quiet` is `true`
-    default: config => ({
-      log: config.quiet ? () => {} : console.log
-    }),
-    legacyName: "logger"
-  },
   verbose: {
     normalize,
     cliDescription: "Set to `true` to log detailed RPC requests.",
     default: () => false,
     legacyName: "verbose",
     cliAliases: ["v", "verbose"],
     cliType: "boolean"
+  },
+  file: {
+    normalize: (raw: PathLike): number => {
+      let descriptor: number;
+
+      try {
+        descriptor = openSync(raw, "a");
```

Comment:
I've added a comment in each of the `logging-options.ts` files.

Originally this was `number | PathLike` (where `number` is a filedescriptor (because sync functions work with number descriptors, and yargs works with async normalisers).

I removed this option, because it makes it hard to decide whether we should close the filedescriptor ourselves or not, and I don't see a huge amount of value to providing this behaviour. If anyone disagrees, I'm happy to look at it again.